.github/workflows/fork.yml

```diff
@@ -0,0 +1,18 @@
+name: "hercules"
+
+on:
+  pull_request:
```

Comment:
This actions isn't sensitive so I think `pull_request` should be fine.

Reply:
It's possible to make the approval optional afaik via project settings.

Reply:
Probably easier to keep this as pull_request_target like the label action.